PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1824,36 +1829,66 @@ void Notepad_plus::filePrint(bool showDialog)
 
 int Notepad_plus::doSaveOrNot(const TCHAR *fn)
 {
-	TCHAR pattern[64] = TEXT("Save file \"%s\" ?");
+	/*TCHAR pattern[64] = TEXT("Save file \"%s\" ?");
 	TCHAR phrase[512];
 	wsprintf(phrase, pattern, fn);
-	return doActionOrNot(TEXT("Save"), phrase, MB_YESNOCANCEL | MB_ICONQUESTION | MB_APPLMODAL);
+	return doActionOrNot(TEXT("Save"), phrase, MB_YESNOCANCEL | MB_ICONQUESTION | MB_APPLMODAL);*/

```

Comment:
Why keep the original source as comment?

Reply:
Because maybe @donho like the old code and `doActionOrNot(...)` function

Reply:
I guess @donho uses a diff/merge tool for that.